tsconfig.json

```diff
@@ -1,25 +1,21 @@
 {
   "compilerOptions": {
-    "target": "es6",
-    "allowJs": true,
-    "lib": ["DOM", "DOM.Iterable"],
-    "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true,
+    "target": "ES2018",
+    "outDir": "dist",
+    "declaration": true,
+    "declarationDir": "dist",
     "esModuleInterop": true,
-    "module": "ES6",
+    "module": "ESNext",
     "moduleResolution": "node",
+    "jsx": "react-jsx",
+    "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "react",
-    "preserveSymlinks": true,
-    "baseUrl": ".",
-    "declaration": true,
-    "outDir": "dist",
+    "skipLibCheck": true,
```

Comment:
What is this for? I'd also try to enable `strict` if possible.

Reply:
on `pnpm` types resolution doesn't work properly without this prop